BossMod/BossModule/AIHints.cs

```diff
@@ -90,16 +90,24 @@ public void FillPotentialTargets(WorldState ws, bool playerIsDefaultTank)
     {
         bool playerInFate = ws.Client.ActiveFate.ID != 0 && ws.Party.Player()?.Level <= Service.LuminaRow<Lumina.Excel.GeneratedSheets.Fate>(ws.Client.ActiveFate.ID)?.ClassJobLevelMax;
         var allowedFateID = playerInFate ? ws.Client.ActiveFate.ID : 0;
-        foreach (var actor in ws.Actors.Where(a => a.Type == ActorType.Enemy && a.IsTargetable && !a.IsAlly && !a.IsDead))
+        foreach (var actor in ws.Actors.Where(a => a.Type is ActorType.Enemy or ActorType.Part && a.IsTargetable && !a.IsAlly && !a.IsDead))
```

Comment:
Is the type check even needed, now that we have IsAlly flag?

Reply:
Oh maybe not. Are eventobjects allies?

Reply:
I think so. Here's the check:
```
        var friendly = chr == null || ActionManager.ClassifyTarget(chr) != ActionManager.TargetCategory.Enemy;
```